.bash_prompt

```diff
@@ -103,14 +103,14 @@ else
 	hostStyle="${yellow}";
 fi;
 
-# Set the terminal title to the current working directory.
-PS1="\033]0; ${PWD##*/}\007";
+# Set the terminal title to the current working folder.
```

Comment:
I have updated the code comments and used base name for just folder name as opposite to full path (based on basename command that extracts the base portion of a pathname)